XenAdmin/Wizards/RollingUpgradeWizard/RollingUpgradeWizardPrecheckPage.cs

```diff
@@ -199,6 +199,11 @@ protected override List<CheckGroup> GenerateChecks(Pool_patch patch)
                 groups.Add(new CheckGroup(Messages.CHECKING_SAFE_TO_UPGRADE, safeToUpgradeChecks));
             }
 
+            //protocol check - for each pool
+            var vswitchChecks = (from Host server in SelectedMasters

```

Comment:
The check is not needed if the current version is Stockholm or greater (as it will always return "OK") so we shouldn't add it here.